### DISCUSSION TITLE
Physics-based density limit

### DISCUSSION THREAD

**Discussion by jonmaddock** [ukaea/PROCESS#3641] [General]
In GitLab by @mkovari on Dec 4, 2019, 10:20

Origin of Tokamak Density Limit Scalings  
D. A. Gates and L. Delgado-Aparicio, Phys. Rev. Lett. 108, 165004 – April 2012

They argue that the density limit is due to radiative effects.  

> For future tokamaks, which may operate at higher temperature and density, bremsstrahlung radiation may become dominant giving a lower density limit. 

This suggests to me that we should be using a physics-based density limit, rather than the empirical Greenwald limit.

Any thoughts - @rkemp @msiccini @skahn ?

PS Michael Fitzgerald's comment - "If someone had figured out the physical origin of the density limit we would have heard about it."

**Comment by jonmaddock** [comment]
In GitLab by @mkovari on May 6, 2022, 16:23

See also [Phys. Rev. Lett. 128, 185003 (2022) - First-Principles Density Limit Scaling in Tokamaks Based on Edge Turbulent Transport and Implications for ITER](https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.128.185003)

A quick reading of this paper seems to show that it is thoroughly dishonest.  Although they say their proposed formula applies to H-mode and L-mode, this isn't true.  Their actual definition of the density limit of discharges with an H-mode phase is as follows.

> the plasma undergoes first an L-H transition, then as the density is increased, plasma confinement degrades until an H-L transition occurs and, once in L-mode, a density limit is attained.

In other words, this is actually an L-mode density limit!

@stuartmuldrew @ajpearcey

**Comment by jonmaddock** [comment]
In GitLab by @mkovari on May 17, 2022, 10:40

*In principle*, their formula could be used for L-mode reactors, and it would allow higher densities for suitable values of the parameters.  However, the issue of confinement degradation with increasing edge density remains.

**Comment by stuartmuldrew** [comment]
Comment from Francis Casson:

>The Greenwald limit is now understood to be a L-mode or separatrix density limit:
>- Most present machines limited by fuelling from SOL
>
>- Reactor devices require core fuelling (pellets)
>
>H-mode operation above Greenwald limit has been demonstrated in devices including DIIID and AUG
>
>- Exact nature of the limit in pellet fuelled reactor plasmas not well understood
>
>- [Ding et al, Nature 2024](https://doi.org/10.1038/s41586-024-07313-3)
>
>- [Manz and Eich, NF 2023](https://doi.org/10.1088/1741-4326/acd9db)

**Comment by stuartmuldrew** [comment]
Given there is no well understood alternative, and the Greenwald fraction is still used as the main density reference, I propose we just close this issue with no change.  It can be moved to a discussion if needed.

**Comment by mkovari** [comment]
I feel I was a bit harsh on the Giacomin paper in PRL.  Fig. 3 shows that both H-mode and L-mode limits follow the scaling.  Equation 12 is actionable:

![Image](https://github.com/user-attachments/assets/e68d728c-eee0-4586-ad51-0de04d3891dd)
q is the edge safety factor at the 95% flux surface.
κ is the plasma elongation but it is not stated whether this is κ_95 or κ_LCFS.

![Image](https://github.com/user-attachments/assets/33ae472e-229e-48ab-a375-7700c776d8f5)
Ptot = total heating power
(PROCESS doesn't follow this exactly convention in the definition of `psep`.)
![Image](https://github.com/user-attachments/assets/f337e3ba-03f0-451b-bda2-5747d37c4a2a)

A number of things would need to be looked at carefully.

> The experimental value of the density at the MARFE onset is obtained by averaging the edge density, measured by means of Thomson scattering, in the region between ρpol = 0.85 and ρpol = 0.95 in all the discharges and tokamaks.

So it isn't the edge density (although they call it that) and it certainly isn't the line-average.

![Image](https://github.com/user-attachments/assets/c5e27b4b-68d3-41bd-b73c-15d3f9014577)
The difference between the edge (ρpol = 1) and ρpol = 0.9 is large.
In fact infinite for the black curve.
 
As Francis mentioned, the limit in pellet-fuelled reactor plasmas is not well understood.  Giacomin says that

> Experiments also demonstrated that the Greenwald limit can be exceeded through pellet injection that mainly increases the core, and only weakly, the edge density.

However, this is for experimental tokamaks.  In a reactor, the pellets will not reach the core and will mainly fuel the edge.

Also, I am not sure how Giacomin has dealt with radiation (see page 3).

We would need to think about how to estimate the volume-averaged or central density from the value at ρpol = 0.9 .

In any case the dependence on P_SOL in equation 12 will be important when extrapolating to a reactor.  It seems sensible to offer this as an option.
